*** task.py	2012-12-18 23:35:28.130310622 +0100
--- task.py.orig	2012-12-18 23:35:20.284309552 +0100
*************** general State and Event.
*** 12,18 ****
  
  """
  
! # from functools import total_ordering
  
  
  class Theme(object):
--- 12,18 ----
  
  """
  
! from functools import total_ordering
  
  
  class Theme(object):
*************** class Theme(object):
*** 30,36 ****
          raise NotImplementedError("Theme needs to be implemented yet.")
  
  
! # @total_ordering
  class Goal(object):
      """
      Goal is a high-level concept, although less abstract than Theme. It is
--- 30,36 ----
          raise NotImplementedError("Theme needs to be implemented yet.")
  
  
! @total_ordering
  class Goal(object):
      """
      Goal is a high-level concept, although less abstract than Theme. It is
*************** class Goal(object):
*** 56,62 ****
          return id(self)
  
  
! # @total_ordering
  class Plan(object):
      """
      Plan is a recipe how to achieve a specified target state from a specified
--- 56,62 ----
          return id(self)
  
  
! @total_ordering
  class Plan(object):
      """
      Plan is a recipe how to achieve a specified target state from a specified
*************** class Plan(object):
*** 91,97 ****
          return id(self)
  
  
! # @total_ordering
  class Task(object):
      """
      Task is an (potentially recurrent) event with an actor, generally one that
--- 91,97 ----
          return id(self)
  
  
! @total_ordering
  class Task(object):
      """
      Task is an (potentially recurrent) event with an actor, generally one that
